integration.js

```diff
@@ -281,6 +281,18 @@ async function onMessage(payload, options, cb) {
         });
       }
       break;
+    case 'GET_WORKFLOW_TEMPLATES':
+      try {
+        const response = await getWorkflowTemplates(options);
+        cb(null, {
+          data: response
```

Comment:
The component will be more readable if you return like this:

```
const worksflows = await getWorkflowTemplates(options);
cb(null, {
  workflows
}
```

Note that you would also need to return `apiResponse.data` from the `getWorkflowTemplate` call.